add ggExtra extension

daattali opened the pull request:
You probably will need to run the makefile and update the rest of the website, this is only the Rmd and its `knitr` outputs

Emaasit commented:
Hi @daattali, I have added ggExtra to the site. Once I have cleaned up the repo, would you be willing to co-maintain the website with me? Let me know.

daattali commented:
Thanks Daniel! Unfortunately really don't have the bandwidth to take on
anything new right now
On 2016-01-31 2:39 AM, "Daniel Emaasit (PhD Student)" <
notifications@github.com> wrote:

> Hi @daattali https://github.com/daattali, I have added ggExtra to the
> site. Once I have cleaned up the repo, would you be willing to co-maintain
> the website with me? Let me know.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ggplot2-exts/ggplot2-exts.github.io/pull/2#issuecomment-177462432
> .
